src/libprojectM/ProjectM.hpp

```diff
@@ -27,12 +27,15 @@
 #include "fatal.h"
 
 #ifdef WIN32
+
+#ifdef _MSC_VER
 // libs required for win32
 #pragma comment(lib, "psapi.lib")
```

Comment:
Not sure these pragmas are required anymore. CMake will probably add the libraries.

Reply:
I think `kernel32.lib` can be removed, but removing `psapi.lib` results in:
```
dlfcn.c.obj : error LNK2019: unresolved external symbol EnumProcessModules referenced in function dlopen
```

That said, I think it would be better to link to `psapi.lib` explicitly only for those who need it. But `target_link_libraries` doesn't work for `OBJECT` targets yet [1], so maybe the iterim solution would be to move this #pragma to the vendored `dlfcn.h`.

[1]. https://gitlab.kitware.com/cmake/cmake/-/issues/18090

Reply:
Could add this to the static/shared library targets, which will also pass the library to downstream users of the CMake package. They'll have a similar issue when linking a static libprojectM.lib into their applications.